conf/swagger.yml

```diff
@@ -6,4 +6,18 @@ paths:
   /RESTAdapter/PLR/UKTaxReturn:
     post:
       security:
-        - {}
\ No newline at end of file
+        - userRestricted: [hello]
```

Comment:
this will change once we have our own scopes: https://github.com/hmrc/api-scope/pull/104

Reply:
I've added these now in preparation for the scopes being created